pallets/ddc-staking/src/mock.rs

```diff
@@ -95,6 +95,7 @@ parameter_types! {
 	pub const DefaultEdgeChillDelay: EraIndex = 1;
 	pub const DefaultStorageBondSize: Balance = 100;
 	pub const DefaultStorageChillDelay: EraIndex = 1;
+	pub const DdcAccountsPalletId: PalletId = PalletId(*b"accounts");
```

Comment:
1. Nice to have it as a constant
2. A bit weird to me to have a hardcoded dependency on another pallet here. Does Substrate have any recommended way of doing this?

Reply:
It's a mock file, dependency is not hardcoded, rather defined in runtime src file, if I understood you correctly